src/elements/content-sidebar/__tests__/DocgenSidebar.test.tsx

```diff
@@ -30,101 +32,72 @@ const defaultProps = {
     ...docGenSidebarProps,
 };
 
-// TODO renable when development returns to normal
-xdescribe('elements/content-sidebar/DocGenSidebar', () => {
-    const getWrapper = (props = defaultProps, options = {}) =>
-        mount(<DocGenSidebar logger={{ onReadyMetric: jest.fn() }} {...props} />, options);
+describe('elements/content-sidebar/DocGenSidebar', () => {
+    const Wrapper = ({ children }: { children?: React.ReactNode }) => {
+        return <IntlProvider locale="en">{children}</IntlProvider>;
+    };
+
+    const renderComponent = (props = defaultProps) =>
+        render(<DocGenSidebar logger={{ onReadyMetric: jest.fn() }} {...props} />, { wrapper: Wrapper });
 
     test('componentDidMount() should call fetch tags', async () => {
-        await act(async () => {
-            await getWrapper(defaultProps);
-        });
-        expect(docGenSidebarProps.getDocGenTags).toHaveBeenCalled();
+        renderComponent();
+        await waitFor(() => expect(docGenSidebarProps.getDocGenTags).toHaveBeenCalled());
     });
 
     test('should render DocGen sidebar component correctly with tags list', async () => {
-        let wrapper;
-        await act(async () => {
-            wrapper = getWrapper(defaultProps);
-        });
-        wrapper!.update();
-        const tagList = wrapper!.find('span.bcs-DocGen-tagPath');
-        expect(tagList).toHaveLength(26);
-        expect(wrapper).toMatchSnapshot();
+        renderComponent();
+        const tagList = await screen.findAllByTestId(/bcs-TagsSection/);
+        expect(tagList).toHaveLength(2);
+        expect(document.body).toMatchSnapshot();
```

Comment:
we should avoid using any snapshots when using RTL and assert based on what the user can see on the screen